cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -902,8 +936,25 @@ Slice::Python::ImportVisitor::addRuntimeImportForMetaType(
 bool
 Slice::Python::PackageVisitor::visitModuleStart(const ModulePtr& p)
 {
-    // Add the __init__.py file to the list of generated modules.
-    _packageIndexFiles.insert(getMappedPackage(p) + "/__init__.py");
+    string packageName = getMappedPackage(p) + p->mappedName();
+
+    // Ensure all parent packages exits, that is necessary to account for modules
```

Comment:
moving this here ensures outer packages are created before the inner packages. which is important for dynamic compilation and avoid the need to sort them later.